plugins/Ecommerce/tests/System/expected/test_ecommerceOrderWithItems_schedrep_html_row_evolution_graph__ScheduledReports.generateReport_week.original.html

```diff
@@ -44,11 +44,6 @@ <h2 id="Actions_getPageTitles" style=" color:#0d0d0d;font-family:-apple-system,
wCwaGQozwOkA5WZnoBoRr+iVOabSlGnRgaA0PkLC8HC+mb+6YlyggaGefMp4lHfJT4+vPQfN+m5lDLkW3qtAKAIH/3y94OF3p4luTNhegIAgE5x+61bTJITNAAA0ClM5iY8ggYAADrHhiGCBgAAIHb58pXHK2wayvkA6QBBAwAANXf67MVgobl6mUk+BA0AANTcg48+GSwsWWJ03idoAACgI7x/93WGORA0AADQEUYG8z8LMkDQAABAR9i03ugqSI+gAQCAeltYvBwsbB5mpAEAAIjNzp0PFlYs6zXMiqABAIA6+/Rf/dDzvK3XD3Z3dxlmRdAAAED9bdk4YJ4JQQMAAPW3edj0KkiPoAEAgE6w2fjWCY+gAQCAGjt/YSFYuJagAQAASHz0y98PFgwfIB0gaAAAoOb27thkJR+CBgAAam7E7BexQwQNAADU3KYh02dBBggaAACoOStXQXoEDQAA1NXc6QvBwuDaFVYyJGgAAKCeHnr0Sc/z3nnDkK0MCRoAAKizDetW2sqKoAEAgDrbOEjQAAAAFIzY+NWJAEEDAAB1pjjS8Ni3t8/+4hfydQgaAACooQOHjgYLq/uXZq782Le3e573jWcnzp48KVmNoAEAgNras13jAdLX/OJtS9eskaxA0AAAQG0NKzyhIRhm8DzvtlvHG729kjUJGgAAqC31+y03/vzWt9x5p3wdggYAAGprZDjjVyfCYYb37T+QmRtBAwAAdTO/cClYGG7KRhpeO3UqWBg9vmds797MbAkaAACom4lHvhcsLOtrSFb76hP7goXbDz6oki1BAwAA9XTne66T/DUcZrju+N6hm9+qkiFBAwAA9SS/dSIcZvjYX3xNMUOCBgAA6mmDwrMg3/byXeoZEjQAAFBPkgdIhzdN3PXIl9UzJGgAAKBWTp+9GCysu2a5fM1bXv2Q/GlOMQQNAADUygN//u+e5+3eNrJkSfpZPhxmeO8nlG6aCBE0AABQQ+tWL0t9f+bFl4KF3acnV6xrauVJ0AAAQA0NrU2/oOHvf/KRYOHdH/+4bp4EDQAA1FDqVZDhxMT+xU8tGxjQzZOgAQCA+rh8uRUsjAwJf3Ximv+5+daH9K5mCBA0AABQH/d//olgYdXKvtifwmGG3Te+P1/mBA0AANTN/p1bRH8aPL7tHQfH82VL0AAAQN0Mrok/oeHqMMP2D+TOlqABAIC6GVr7hqBh/uy5YGHj8XffdO/dubMlaAAAoG42DL7hKsivfGdPsHDbXR8zyZagAQCAmlhYvBwsrF+Xcr/lxp/v3PyuW0zyJ2gAAKAmDn7xu8HCsr5G+GZ4NcMDj/6tYf4EDQAA1MoH94wl33zz1H6t36ZKRdAAAECtDEZ+3DIcZrj7U58zz5mgAQCAWhluroi9c9MvP7hyeNg8Z4IGAADq4OoDpIdXBQvhMMPvf+wBK0UQNAAAUAfhA6TXDizzIhHDtpP3XzO6xUoRBA0AANTHe96xsadx9eS+/OWxvZMP2MqcoAEAgPpYt2aFFxlmuHnZDitXMwQIGgAAqI/1kQdIr/zV793x+S9YzJygAQCA+li/rj8cZti5fp/dzAkaAACovPAB0iPDV351Ys3U23f90UN2SyFoAACg8sIHSH/9O7uChV037LdeCkEDAAA18d6tQ8HC4PHt7/zwhPX8CRoAAKiJU95ng4Xb/sB+xOARNAAAUDPN4+8c27u3iJwb0RdjY1d/F2tqaqqI8gAAQBHevvUvg4U//PCfFlTE1aBhbGwsGijEXgIAADe9dnExXN58/D3Dk28pqCCmJwAAqLaP/Nm/hcMM7z/4x8UVRNAAAEBNvGlq39DNby0u/0b2Kpr+8Uf3WM8TAACIvH3rlYX3HfhIoQVlBw0//elPoy9ffPHFG2+8sbD6AACAPDb+bO9vblr4zXPPFVdEdtCwY8cOycuknxy55VL34vLWSpNqnRx8adXMxqWtFUaZbPrZ4K+N4ptzjVMXBubWzbzJJJNTK05eWrLYPLPJJJOZa070nusfmG+aZGKrVZu/ektXV/6JrfM9tGoiE1r1jWjVGFo1pk6tunhpcWbLC4MvG52tLnUtLvEatz/w0KabbzDJJ5P96Yk7P/nF5cuXDw0NWc8ZsOvEiRP0VVQCfRWOuBo0TE1N8ZwGAAAg8oaRBgIFAAAgwi2XAABASVer1Sq7DgAAoAIanuAnJ+69997UBMePH3/hhRckOSomDAqNPbg6WQ2R1JVFORiWBRWpjZy7bdX3b6EdVaubpaKjop14/D+K1pD85MQTTzyRTCA61EZlJgxKiR4QtX75InVlUQ6GZaH9tPavV1hH1a2G4YbQUYtj2HTqQV5xIWy0Dp5xBBnmVnTUK3oT1dUoZV8aFqqVnM7aNskzYiB5iImulkySusvavx/NS6Sj1oB6kBco7ruWrR6iHkIZRr2iN1Fp9p/TAETxdRlOiX3Jjp3Vkp1T1F1d6Mbyb/xeWiW1PoDmUa8LrQS7rgQN6qNVruEk5BTRYINotcyVRUMXbd7pYYm5H2dCR3VEcohLPu7lLPk3/tR1wjfD5apsLNxxJWjI8ZlJHXHK3QVzDGExVeYmxbghk635UcOOGisxx8AJHdVxKrvGha9Viid7lZA9mme+qQp0rPzTE9Y7kOii8VT04HqzOMBgeAWc+RVndFSn5JtftzUUYRLCyscMikPUi6j43RMuUBlec63OiFIZN5IMSDiy07VKdKTOkIsdW9pfAbv9oQ29iz6MmIZkjjb1it/jx49nZpqZMCwxGsNqDQAmb0ASbUiyrNzT0sghX2un3mAmuuusDR01LFF3c+ioVVeVs6b5EAhRL1S4/kRIei0qgY7qMtEAQ3K4S/3KwdQQttDnNKhUTOVxDqLNUZn+SJYibxCV+qBanA4aOBCjEuiojmMHZaKJoMjpH6yiE6MS6KioNCIGqHN6pAEATHDlP2AXQQMAAFDi9PQEAABwB0EDAABQ8v8IS5DCFDTUzgAAAABJRU5ErkJggg=="
```

Comment:
@diosmosis are you able to explain why this test file changes are expected? Wondering why the images shouldn't be included here anymore

Reply:
I'm not sure, I'll take a look.

Reply:
@sgiehl it doesn't look like those tests are run. They're not for me locally at least. It might be that `Fixture::canImagesBeIncludedInScheduledReports()` may need to be adjusted after the move to github actions.

Reply:
Ok. That's indeed something we need to investigate. But that's unrelated to this PR then.